src/main.rs

```diff
@@ -189,7 +190,7 @@ fn install(args: InstallOpts) -> Result<()> {
 
     exports.extend(llvm.install()?);
 
-    if targets.contains(&Target::ESP32C3) {
+    if targets.contains(&Target::ESP32C3) || targets.contains(&Target::ESP32C2) {
```

Comment:
Maybe a nicer way of handling this (which could be done in a separate PR) would be to implement some helpers on `Target` to determine the architecture. As we continue to add support for addition devices we're going to keep needing to chain expressions with `||` here.

Maybe if we do something like this:

```rust
impl Target {
    pub fn riscv(&self) -> bool {
        !self.xtensa()
    }

    pub fn xtensa(&self) -> bool {
        matches!(self, Target::Esp32 | Target::Esp32s2 | Target::Esp32s3)
    }
}
```

We can then change this line (and the others like it) to something like this:

```rust
if targets.iter().any(|t| t.riscv()) {
    //
}
```

Just a thought, doesn't really matter right now but will likely clean things up in the future. If you don't like this idea then that is fine of course too 😁